custom_addons/non_conformance_report/models/x_ncr_report.py

```diff
@@ -60,18 +59,31 @@ def create(self, vals_list):
             ('rejected', 'Rejected'),
             ('return_for_further_actions', 'Return for Further Actions'),
             ('closed', 'Closed')
-        ], tracking=True, default ='new'
+        ], tracking=True, default='new', string='Status'
         # Set a default value for the state field
     )
 
     due_date = (datetime.now() + timedelta(days=2)).strftime('%Y-%m-%d')
 
+    @api.onchange('ncr_initiator_id')
+    def _onchange_ncr_initiator_id(self):
+        if self.ncr_initiator_id:
+            if self.ncr_initiator_id:
+                self.initiator_job_title = self.ncr_initiator_id.employee_id.job_id.name if self.ncr_initiator_id.employee_id else "No Job Title"
+            else:
+                self.initiator_job_title = "No Job Title"
+        else:
+            self.initiator_job_title = False
+
     @api.onchange('ncr_initiator_id')
     def _set_approver_id(self):
-        if self.ncr_initiator_id.parent_id:
-            self.ncr_approver_id = self.ncr_initiator_id.parent_id
+        if self.ncr_initiator_id:
```

Comment:
Here also you need to do null check for parent_id. Just in case no manager is assigned and also null check for employee id